README.md

```diff
@@ -10,6 +10,15 @@ Include the entire thing as a CocoaPod by referencing it on GitHub or as a submo
 
 After installing LiveFrost, place `LFGlassView` in a view you’d like to blur and blurring will commence automatically. Use `LFGlassView` as any other `UIView`.
 
+
+## UserInteractionEnabled
+By default userInteractionEnabled is NO, so touch events will be handled by the UIView underneath your LFGlassView. To prevent touch events from being sent to the view under your LFGlassView:
+
+```
+LFGlassView glass = [[LFGlassView alloc] init];
```

Comment:
The pointer sign `*` should be placed in front of `glass`:
`LFGlassView *glass = [[LFGlassView alloc] init];`
This should compile